src/framework/preferences/css/stylus/README.md

```diff
@@ -4,25 +4,21 @@ The "[stylus](./)" directory contains Stylus files for generating the Preference
 
 ## How to add a new theme
 
-Define your own themes variable that looks like:
+Define your own theme variables in `utils/Themes.styl` using the following approach.
 
 ```stylus
-    contrastThemes = {
-        "theme-selector": {
-            foregroundColor: "#000000",
-            backgroundColor: "#ffffff"
-        }
-        // ...
+    .fl-theme-selector { // The CSS class that the Preferences Framework uses to enable this contrast theme.
```

Comment:
I'm not sure that it's clear here that `.fl-theme-selector` should be replaced with the name of your own theme, and that `fl-theme-` is the conventional prefix with the `selector` part being replaced with the specific theme name.